animals_and_tips.html

```diff
@@ -126,108 +126,115 @@ <h2>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod dog<
 <div class="container">
 	<img src="images/placeholder.png" alt="img" class="image">	
 	<div class="overlay overlayLeft">
-		<div class="text"><h1>Header</h1><h2>Subheader</h2><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco<p></div>
+		<div class="text"><h1>Lion</h1><h2>Panthera Leo</h2><p>Stand your ground, retreating very slowly, continuing to face the lion while clapping your hands, shouting and waving your arms around to make yourself look bigger. Most charges are mock charges, so you will usually be fine. And hold your ground! Never run or turn your back.

```

Comment:
Information about certain animals were added, with their name, scientific name and information about such as if they are dangerous and what to do if you encounter them